table/column.go

```diff
@@ -152,7 +155,8 @@ func CastValues(ctx sessionctx.Context, rec []types.Datum, cols []*Column) (err
 func handleWrongUtf8Value(ctx sessionctx.Context, col *model.ColumnInfo, casted *types.Datum, str string, i int) (types.Datum, error) {
 	sc := ctx.GetSessionVars().StmtCtx
 	err := ErrTruncateWrongValue.FastGen("incorrect utf8 value %x(%s) for column %s", casted.GetBytes(), str, col.Name)
-	log.Errorf("con:%d %v", ctx.GetSessionVars().ConnectionID, err)
+	conn := uint32(ctx.GetSessionVars().ConnectionID)
+	zaplog.Error("incorrect UTF-8 value", zap.Uint32("conn", conn), zap.Error(err))
```

Comment:
Why not use `logutil.Logger`?

Reply:
Because we have no valid `context.Context` parameter here, if we use `logutil.Logger`, it would generate a field like `conn: ''` by default, and then if we add `zap.Uint32("conn", conn)`, we would end up with two fields of `conn`, one is empty, and the other is the valid connection id. It may look weird, so I use zaplog directly here.

Reply:
Yes, and it is the same as other places that use `context.Background`. According to the discussion with @XuHuaiyu, currently, we will use `logutil.Logger` even if there is no valid context.

Reply:
Alright, code updated, PTAL